packages/node-http-handler/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Why is `tsconfig.json` required in node-http-handler?

Reply:
good catch. Will remove